### PR TITLE
remove ODE.jl, update solver versions

### PR DESCRIPTION
**Pull request by oscardssmith**

### CODE REVIEW

File: benchmarks/NonStiffODE/LotkaVolterra_wpd.jmd [hunk 1]
```diff
@@ -9,7 +9,7 @@ author: Chris Rackauckas
 The purpose of this problem is to test the performance on easy problems. Since it's periodic, the error is naturally low, and so most of the difference will come down to startup times and, when measuring the interpolations, the algorithm choices.
 
 ```julia
-using OrdinaryDiffEq, ParameterizedFunctions, ODE, ODEInterfaceDiffEq, LSODA,
```
Comment (ChrisRackauckas) [inline benchmarks/NonStiffODE/LotkaVolterra_wpd.jmd:12]: won't you need to remove the solvers that use it too?

Comment (oscardssmith) [reply benchmarks/NonStiffODE/LotkaVolterra_wpd.jmd:12]: I believe all the solvers we are using from it are in ODEInterfaceDiffEq (but I'll double check).

**Comment by oscardssmith** [comment]
it looks like the MTK v9 upgrade to the stiff case doesn't work automatically...

**Comment by oscardssmith** [comment]
This seems to have worked now!

File: benchmarks/StiffODE/VanDerPol.jmd [hunk 2]
```diff
@@ -139,7 +139,7 @@ setups = [Dict(:alg=>Rosenbrock23()),
           Dict(:alg=>Rosenbrock23(), :prob_choice => 2),
           Dict(:alg=>Rodas3()),
           Dict(:alg=>TRBDF2()),
-          Dict(:alg=>rodas()),
```
Comment (ChrisRackauckas) [inline benchmarks/StiffODE/VanDerPol.jmd:142]: why? We want to benchmark against Hairer.

File: benchmarks/StiffODE/VanDerPol.jmd [hunk 3]
```diff
@@ -320,7 +319,6 @@ setups = [Dict(:alg=>Rodas3()),
           Dict(:alg=>CVODE_BDF()),
           Dict(:alg=>Rodas4()),
           Dict(:alg=>Rodas4(), :prob_choice => 2),
-          Dict(:alg=>rodas()),
```
Comment (ChrisRackauckas) [inline benchmarks/StiffODE/VanDerPol.jmd:323]: why remove Hairer?

**Comment by ChrisRackauckas** [comment]
Rosenbrock static took a major performance hit?

File: benchmarks/StiffODE/VanDerPol.jmd (hunk 2 above)
Comment (oscardssmith) [reply benchmarks/StiffODE/VanDerPol.jmd:142]: it was hanging

Comment (ChrisRackauckas) [reply benchmarks/StiffODE/VanDerPol.jmd:142]: Add a note about that. There's always a notes on why some methods weren't benchmarked.

**Comment by oscardssmith** [comment]
How are you looking at the results?

**Comment by oscardssmith** [comment]
To me it looks like the static rosenbrocks are faster?

**Comment by ChrisRackauckas** [comment]
See Hires

**Comment by ChrisRackauckas** [comment]
But add the note so we can merge.

**Comment by oscardssmith** [comment]
note added.

**Pull request merged**